ufc/tests/test-case-flag-with-empty-string.json

```diff
@@ -0,0 +1,107 @@
+{
+  "flag": "string-variations",
+  "variationType": "STRING",
+  "defaultValue": "default",
+  "subjects": [
+    {
+      "subjectKey": "alice",
+      "subjectAttributes": {},
+      "assignment": "",
+      "evaluationDetails": {
+        "environmentName": "Test",
+        "flagEvaluationCode": "MATCH",
+        "flagEvaluationDescription": "alice belongs to the range of traffic assigned to \"empty_string\" defined in allocation \"allocation-empty\".",
+        "banditKey": null,
+        "banditAction": null,
+        "variationKey": "empty_string",
+        "variationValue": "",
+        "matchedRule": null,
+        "matchedAllocation": {
+          "key": "allocation-empty",
+          "allocationEvaluationCode": "MATCH",
+          "orderPosition": 1
+        },
+        "unmatchedAllocations": [],
+        "unevaluatedAllocations": []
+      }
+    },
+    {
+      "subjectKey": "bob",
```

Comment:
The assignments for subjects bob, charlie and david  are invalid - the expected `assignment` must be a string.